rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -611,6 +605,26 @@ public String getNodeMBeanHistory(@HeaderParam("sessionid") String sessionId,
         return rmProxy.getNodeMBeanHistory(nodeJmxUrl, objectName, attrs, range);
     }
 
+    @Override
+    @GET
+    @GZIP
+    @Produces("application/json")
+    @Path("nodes/mbean/history")
+    public String getNodesMBeanHistory(@HeaderParam("sessionid") String sessionId,
+            @QueryParam("nodejmxurl") Set<String> nodesJmxUrl, @QueryParam("objectname") String objectName,
+            @QueryParam("attrs") List<String> attrs, @QueryParam("range") String range)
+            throws InstanceNotFoundException, IntrospectionException, ReflectionException, IOException,
+            NotConnectedException, MalformedObjectNameException, NullPointerException, MBeanException {
+
+        // checking that still connected to the RM
```

Comment:
//checking that still connected to the RM to remove it was previously explained in other endpoints

Reply:
ok